### DISCUSSION TITLE
社交登录弹窗未显示

### DISCUSSION THREAD

**Discussion by gzzzxx** [ArtalkJS/Artalk#888] [Q&A] [answered]
请大佬帮忙解答，开启了社交登录，点击登录为什么是这样的
![image](https://github.com/ArtalkJS/Artalk/assets/36790176/7ebd03ab-0295-452b-854b-78f20d9ee0e8)

**Comment by qwqcode** [comment]
试试升级前端 JS 到最新版

**Comment by gzzzxx** [comment]
docker run -d --name artalk -p 8090:23366 -v $(pwd)/data:/data --restart=always artalk/artalk-go:nightly
我是用这个命令部署的，怎么单独升级前端JS，麻烦大佬指教

**Reply by qwqcode** [reply, marked as answer]
如果是用的 nightly 版，可以在 Release 页面下载 artalk-ui 压缩包里面的 JS 资源然后替换旧版前端 JS，或者参考以下文档：

https://artalk.js.org/guide/backend/fe-control.html#%E5%9C%A8%E5%89%8D%E7%AB%AF%E5%BC%95%E5%85%A5%E5%86%85%E7%BD%AE%E8%B5%84%E6%BA%90

直接引用服务器内置的 JS

**Comment by qwqcode** [comment]
你先看看你前端是最新版吗，会不会是缓存没刷新

**Comment by gzzzxx** [comment]
这看起来应该不是最新的
![image](https://github.com/ArtalkJS/Artalk/assets/36790176/6a70d72f-3124-4459-83fc-4c0d53198fca)
